test/db/tools/ravc2

```diff
@@ -0,0 +1,76 @@
+NAME=ravc2 init
+FILE=-
+CMDS=<<EOF
+mkdir ravc2_test
+cd ravc2_test
+!!ravc2 init
+lr
+!!ravc2 status
+!!cd ../
+EOF
+EXPECT=<<EOF
+./.rvc
+./.rvc/commits
+./.rvc/blobs
+./.rvc/branches.sdb
+Branch: master
+All files are committed
+EOF
+RUN
+
+NAME=ravc2 commit
+CMDS=<<EOF
+mkdir ravc2_commit_test
+cd ravc2_commit_test
+!!ravc2 init
+echo hello > hello
+!!ravc2 commit "hello world!" hello
+!!ravc2 log~!author,time,hash
+cd ../
+EOF
+EXPECT=<<EOF
+message=hello world!
+----
+/hello=5891b5b522d5df086d0ff0b110fbd9d21bb4fc7163af34d08286a2e846f6be03
+****
+EOF
+RUN
+
+NAME=ravc2 branch
+CMDS=<<EOF
+mkdir ravc2_branch_test
+cd ravc2_branch_test
+!!ravc2 init
+!!ravc2 branch hello
+echo hello > hello
+!!ravc2 commit "hello world!" hello
+!!ravc2 checkout hello
+ls
+!!ravc2 status~Branch:
+!!ravc2 checkout master
+ls
```

Comment:
you can use ls -q for one file per line which is more readable

Reply:
yes, but the reason tests are failing is because the entries are sorted differently.